web/admin/application/configs/klear/model/Friends.yaml

```diff
@@ -344,6 +344,23 @@ production:
         values:
           'yes': _('Yes')
           'no': _('No')
+    alwaysApplyTransformations:
+      title: _('Always apply transformations')
+      type: select
+      defaultValue: 0
+      source:
+        data: inline
+        values:
+          '0':
+            title: _("No")
+          '1':
+            title: _("Yes")
+      info:
+        type: box
+        position: left
+        icon: help
+        text: _("If set to No, numeric transformation won't apply for numbers in Extensions or numbers matching any Friend regexp. If Yes, even those number will traverse numeric transformations rules.")
```

Comment:
```suggestion
        text: _("Enable to force numeric transformation on numbers in Extensions or numbers matching any Friend regexp. Otherwise, those numbers won't traverse numeric transformations rules.")
```